multimodal/src/autogluon/multimodal/models/ner.py

```diff
@@ -0,0 +1,265 @@
+import logging
+from typing import List, Optional, Tuple
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+from transformers import AutoTokenizer, T5TokenizerFast
+from transformers import logging as hf_logging
+from transformers.models.t5 import T5PreTrainedModel
+
+from ..constants import (
+    AUTOMM,
+    COLUMN,
+    COLUMN_FEATURES,
+    FEATURES,
+    LABEL,
+    LOGITS,
+    MASKS,
+    NER_ANNOTATION,
+    TEXT_SEGMENT_IDS,
+    TEXT_TOKEN_IDS,
+    TEXT_VALID_LENGTH,
+    TOKEN_WORD_MAPPING,
+    WORD_OFFSETS,
+)
+from .utils import DummyLayer, assign_layer_ids, get_column_features, get_hf_config_and_model, init_weights
+
+hf_logging.set_verbosity_error()
+
+logger = logging.getLogger(AUTOMM)
+
+
+class HFAutoModelForNER(nn.Module):
```

Comment:
Can we inherit https://github.com/awslabs/autogluon/blob/c1e1eb5fe6eabac739dcf5019f1e25c5cd4e1979/multimodal/src/autogluon/multimodal/models/huggingface_text.py#L28 ?